book/website/collaboration/ppie-events/ppie-events-socials.md

```diff
@@ -0,0 +1,124 @@
+
+For example, poster sessions at research events are often considered to involve short interactions between a researcher and a group of people. 
+At a poster session, an early career researcher may have the opportunity to share and explain their research project. 
+These sessions would not be the best forum for public discourse about an attendee's lived experience or for criticizing the purpose of the work that is being presented.
+Instead, poster sessions may be useful to make introductions with other attendees who are interested in a particular topic, and are an opportunity to learn about the research that has been conducted by asking questions about the research that person is presenting on their poster.
+
+Depending on the audience, your guide may need to be more or less detailed.
+By providing guidance on the type of sessions, and what is expected in each setting, you can help facilitate more constructive and collaborative interactions between attendees, and reduce the likelihood of misunderstandings.
+
+(cl-ppie-events-conversation-prompts)=
+### Sparking conversation between attendees 
+Social skills are fundamental in building relationships, sharing ideas, and collaborating on research endeavours.
+
+Providing guidance on how to start conversations and socialize effectively can help support attendees with lived experience in developing professional connections and meaningfully engage in your event.
+
+Depending on the size and timings of your event, you can consider setting prompts such as "find a fellow attendee who has the same favourite movie as you", or "introduce yourself to 5 people you have never met before, and find 1 thing you have in common". 
+Prompts such as these can provide attendees with a goal for the session and help attendees to start conversations with one another. 
+
+Popular conversation starters can also be providing a blank space or additional sticker for attendees' name tags with the prompt "Ask me about ...". 
+This can help attendees find conversations with similar interests and can also help to spark initial conversations. 
+
+Good conversation prompts or ice breaker questions are both interesting for someone to think about and answer themselves, and also make attendees curious to hear everyone else’s answer.
+
+Knowing how to start a conversation and navigate social interactions can help boost attendees' confidence and support them in actively engaging in any social or networking activities.
+
+(cl-inclusive-events-introduction-guide)=
+#### How to introduce yourself 
+You may consider providing examples of how to introduce yourself and start conversations. 
+Pop-culture references providing good examples of how to introduce yourself can be found in the 1987 movie [The Princess Bride](https://en.wikipedia.org/wiki/The_Princess_Bride_(film)) featuring a character called Inigo Montoya, or Pixar animation's movie [Up](https://en.wikipedia.org/wiki/Up_(2009_film)) featuring a character named Russell.
```

Comment:
I love that so much! ❤️

Reply:
more seriously: maybe worth to link to the memes in questions somewhere though? otherwise it remains a cute in-joke

Reply:
Done! I added youtube links but can go to still images instead. 

Do you think maybe embedding images may be better?

Reply:
I think the YouTube links are great, thanks for adding those!